From 4eeb3bc888b34da7f0749e51366b5190d0ae0e90 Mon Sep 17 00:00:00 2001
From: Cristian Ciocaltea <cristian.ciocaltea@collabora.com>
Date: Mon, 13 Mar 2023 11:53:05 +0200
Subject: [PATCH 433/469] arm64: dts: rockchip: rk3588: Add I2S nodes

In addition to the five I2S/PCM/TDM controllers and the two I2S/PCM
controllers shared between the RK3588 and RK3588S SoCs, RK3588 contains
another four I2S/PCM/TDM controllers.

Add the DT nodes corresponding to the additional controllers.

Signed-off-by: Cristian Ciocaltea <cristian.ciocaltea@collabora.com>
---
 arch/arm64/boot/dts/rockchip/rk3588.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3588.dtsi b/arch/arm64/boot/dts/rockchip/rk3588.dtsi
index 40fee1367b34..28013c23c55a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3588.dtsi
@@ -132,6 +132,7 @@ pcie2x1l0_intc: legacy-interrupt-controller {
 			interrupt-parent = <&gic>;
 			interrupts = <GIC_SPI 240 IRQ_TYPE_EDGE_RISING 0>;
 		};
+
 	};
 
 	gmac0: ethernet@fe1b0000 {
-- 
2.34.1

